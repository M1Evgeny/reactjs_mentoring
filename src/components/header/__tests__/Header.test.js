import React from "react";
import { render } from "@testing-library/react";
import { Header } from "..";

describe("Header test", () => {
  test("Header snapshot", () => {
    const { asFragment } = render(<Header />);
    expect(asFragment(<Header />)).toMatchSnapshot();
  });
});