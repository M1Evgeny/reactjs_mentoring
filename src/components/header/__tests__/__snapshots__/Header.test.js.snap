// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header test Header snapshot 1`] = `
<DocumentFragment>
  <header
    class="header"
  />
</DocumentFragment>
`;
